src/SSHConfig.ts

```diff
@@ -0,0 +1,142 @@
+import { ensureDir } from "fs-extra"
```

Comment:
What do you think about camel casing this name instead? So there's uniformity with other files.

Reply:
I just started it with capital case because it is a class, but I can do it.